mne/epochs.py

```diff
@@ -1546,6 +1549,34 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             epochs.drop_bad_epochs()
         # figure out how to equalize
         eq_inds = list()
+
+        # deal with hierarchical tags
+        ids = epochs.event_id
+        if "/" in "".join(ids):
+            event_ids = [[x] if isinstance(x, string_types) else x
+                         for x in event_ids]
+            event_ids = [[k for k in ids if all((tag in k.split("/")
+                         for tag in id_))]
+                         if all(id__ not in ids for id__ in id_)
+                         else id_
+                         for id_ in event_ids]
+            for id_ in event_ids:
+                if len(set([sub_id in ids for sub_id in id_])) != 1:
+                    err = ("Don't mix hierarchical and regular event_ids"
+                           " like in \'%s\'." % ", ".join(id_))
```

Comment:
Note: if you're a better programmer than I am, you could probably easily set it up so that this is indeed possible.